### DISCUSSION TITLE
Output Selection to stdout

### DISCUSSION THREAD

**Discussion by davidturissini** [ankitpokhrel/jira-cli#636] [Ideas]
I think it would be really nice if I could select an issue from the interactive view and have the jira id outputted to STDOUT. Right now, it seems that you can only open the web browser after selecting and issue. If I could "select" an issue and have that be pipe-able, it would really open the possibilities.

Example:
```
$ jira sprint list --current | xargs echo # FOO-1234
```

More specifically:
```
$ jira sprint list --current | xargs -I '{}' git checkout -b "{}"
```

**Comment by ankitpokhrel** [comment]
@davidturissini Not sure if I understood this properly. Do you mean to do something like this?

```sh
jira sprint list --current --plain --no-headers --columns=KEY | xargs echo
```

**Reply by davidturissini** [reply]
Yes, but instead of just piping the whole list, I want to select a specific issue using the existing views.

The flow is:
- `$ jira sprint list --current`

- Select an issue from the list

- that selected issue is piped to stdout.

As far as I can tell, you can only "select" an issue to open in browser

**Reply by stubar** [reply]
I would also find this useful

**Comment by pataquets** [comment]
I use the following command to create a preview-capable issue selector using [fzf](https://github.com/junegunn/fzf):
```bash
jira issue list --plain --no-headers --columns key,summary | \
fzf --track --no-separator --info=inline-right \
  --preview='jira issue view {1} --comments=999' \
  --preview-window='up:80%,border-bottom' | \
cut -f 1
```
